tests/extras/datasets/api/conftest.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+import requests_mock
+
+
+@pytest.fixture
+def requests_mocker():
```

Comment:
Is there a difference between defining your own fixture and using the provided one from `requests_mock`? 
https://requests-mock.readthedocs.io/en/latest/pytest.html

You could just use `requests_mock` as a fixture?

Reply:
good one!